api/pkg/runner/slot.go

```diff
@@ -127,6 +209,23 @@ func (s *Slot) Create(ctx context.Context) (err error) {
 	if err != nil {
 		return
 	}
+
+	// For VLLM runtime, skip model verification since it starts with the specified model
```

Comment:
No you should still do the list models call, because someone might call this with a different model name.

You can call independently of helix remember.

Reply:
I couldn't get it working though.